### PR TITLE
Update trip viewer semantic HTML

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
Upgrading the semantic HTML of the Trip Viewer for a11y purposes:
- Change Trip Viewer header to `h1`
- Change route name to `h2`
- Remove h4, and conditionally render wheelchair and bike accessibility container
- Add styled components to avoid inline styles
- Group stops in a `ul`

### CODE REVIEW

**Review by binh-dam-ibigroup — APPROVED** [review]
Looks good for the stated scope (see non-blocking comments).

File: lib/components/viewers/trip-viewer.js [hunk 1]
```diff
@@ -87,89 +102,94 @@ class TripViewer extends Component {
...
-                )}
-              </h4>
+              {(tripData.wheelchairAccessible === 1 ||
+                tripData.bikesAllowed === 1) && (
+                <div>
+                  {tripData.wheelchairAccessible === 1 && (
+                    <BsLabel bsStyle="primary">
+                      <IconWithText Icon={Wheelchair}>
+                        <FormattedMessage id="components.TripViewer.accessible" />
+                      </IconWithText>
```
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:115]: Note (TODO): the labeling of the wheelchair/bike badges will need to be redone. The labeling of these badges is currently incomplete for screen readers.

File: lib/components/viewers/trip-viewer.js [hunk 2]
```diff
@@ -87,89 +102,94 @@ class TripViewer extends Component {
...
+                      <div className="strip-map-icon">
+                        <StyledIconWrapper>
+                          <Circle />
+                        </StyledIconWrapper>
+                      </div>
+                    </div>
+
+                    {/* the stop-viewer button */}
+                    <div className="stop-button-container">
+                      <ViewStopButton
```
Comment (binh-dam-ibigroup) [inline lib/components/viewers/trip-viewer.js:177]: Note (TODO): To a screen reader, "View" appears before the stop name. A separate label for screen readers will need to be set up.

File: lib/components/viewers/trip-viewer.js [hunk 3]
```diff
@@ -22,6 +24,20 @@ import ViewStopButton from './view-stop-button'
 
 const { getCurrentDate } = coreUtils.time
 
+const StopList = styled.ul`
```
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply lib/components/viewers/trip-viewer.js:27]: Can't hurt to keep it consistent with #736!

**Review by miles-grant-ibigroup — APPROVED** [review]
Don't think it's worth being too picky as we're probably going to re-write/deprecate this view soon anyway.

Approved pending the Percy tests passing -- I think that selector just needs to be adjusted. Feel free to chat with me offline if my suggestion doesn't work

**Pull request merged**